css/base.css

```diff
@@ -0,0 +1,60 @@
+body {
+  background-image: url(/Users/kavdi/Codefellows/CF-301d24/portfolio/img/cityStreets.jpg);
```

Comment:
This needs to be a relative URL in order for this to work anywhere besides your computer.